src/main/java/nl/thehyve/ocdu/models/OcDefinitions/MetaData.java

```diff
@@ -96,4 +99,11 @@ public MetaData() {
         this.itemGroupDefinitions = new ArrayList<>();
         this.eventDefinitions = new ArrayList<>();
     }
+
+    public boolean isRequired(String attr) {
```

Comment:
Are those remains of a previous approach? If yes - please remove it. If no - please explain how it is supposed to work.